src/common/readout/ess/Parser.cpp

```diff
@@ -14,8 +14,8 @@
 
 namespace ESSReadout {
 
-// #undef TRC_LEVEL
-// #define TRC_LEVEL TRC_L_WAR
+#undef TRC_LEVEL
+#define TRC_LEVEL TRC_L_WAR
```

Comment:
Is this change for good reason or it left after test and debug?

Reply:
Yes, it was on purpose, but I suppose that this is something we should discuss.